src/components/Header/Menu/LeftMenu.js

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { useState } from 'react';
+import { TiThMenu } from 'react-icons/ti';
+import styled from 'styled-components';
+import ProjectModal from '../../Modal/ModalProject/ModalProjectPage';
+
+function LeftMenu() {
+  const [modalOn, setModalOn] = useState(false);
+
+  const onOpenModal = () => {
+    setModalOn(!modalOn);
+  };
+
+  const Modal = () => {
+    return <ProjectModal onClick={onOpenModal} />;
+  };
+  return (
+    <Left>
+      <ProjectLooKAround onClick={onOpenModal}>
+        <IconBox>
+          <TiThMenu />
+        </IconBox>
+        <Title>프로젝트 둘러보기</Title>
+      </ProjectLooKAround>
+      {modalOn ? <Modal /> : ''}
```

Comment:
위 리뷰와 연계하여 아래와 같이 하면 좋을 것 같습니다.
```suggestion
      {modalOn && <ProjectModal onClick={onOpenModal} />}
```